.github/workflows/validations.yml

```
@@ -3,101 +3,332 @@ on:
+          timeout 30 bin/vite build || echo "Skipping vite build check"
+      
+      - name: Install Playwright Chromium browser (with deps)
+        if: steps.playwright-cache.outputs.cache-hit != 'true'
+        run: pnpm exec playwright install --with-deps chromium
+      
+      - name: Install Playwright Chromium browser deps only
+        if: steps.playwright-cache.outputs.cache-hit == 'true'
+        run: pnpm exec playwright install-deps chromium
+      
+      - name: Run Rubocop
+        if: matrix.ci_node_index == 0
```

actionlint (1.7.10)

[error] property "ci_node_index" is not defined in object type {}

(expression)